Ports/lxt/package.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=lxt
+version=1.3b
+files="http://stahlke.org/dan/lxt/lxt-${version}.tar.gz lxt-${version}.tar.gz 7a3ab299a6d61a71b271fd13b847b5a1c22a5f95df78561a325c78d50b6a6bc7"
+useconfigure='true'
+auth_type="sha256"
+depends=('ncurses' 'bash')
+
+pre_configure() {
+    # Rebuild after patching configure.ac to support serenity host.
+    # `automake` may exit with a warning about how there is a mismatch
+    # between the versions of autoconf and automake that were previously
+    # used to generate aclocal and specifed in configure.ac.
+    # We just need `automake` to generate `./compile` (so that we can run
+    # autoreconf to regenerate everything).
+    run aclocal
+    run automake --add-missing
+    run autoreconf
```

Comment:
We should just patch `configure` directly, or properly fix whatever it tries to query. `autoconf` and friends are annoying to work with because they make the resulting configure script dependent of the used version of `autoconf`.

Reply:
From what I see with some additional debugging, the issue is caused by a botched check with the fnmatch function when crosscompiling. Apparently it thinks it's broken while we're crosscompiling (albeit the configure log clearly says "checking for working POSIX fnmatch: cross" and attempts to symlink a nonexistent header to use during config.status phase.

This specific reconfiguration code was copy-pasted from another port and slightly adapted to work with this package IIRC (I wrote this port on 8th of March) and I have no idea how to fix it in configure. Also how would someone have two versions of autoconf with exactly the same name where one will be used to regenerate the configure script and another will actually be seen by configure?